atc/user.go

```diff
@@ -0,0 +1,13 @@
+package atc
+
+import (
+	"time"
+)
+
+type User struct {
+	ID        int       `json:"id,omitempty"`
+	Username  string    `json:"username,omitempty"`
+	Connector string    `json:"connector,omitempty"`
+	LastLogin time.Time `json:"last_login,omitempty"`
```

Comment:
We are using int64 for timestamp in api for example [build](https://github.com/concourse/concourse/blob/5824dfd73cba6a3d9a256a2317f45a92478aa2b4/atc/build.go#L22). Could we make it the same for `LastLogin`? I think it will simplify some tests that related to timestamp as well.

Reply:
Latest commit changes the type to int64 now.